README.md

```diff
@@ -940,6 +941,26 @@ The answer is `3`. Surprised?
 
 * [**13.15** The `try` Statement](https://www.ecma-international.org/ecma-262/#sec-try-statement)
 
+## Asynchronous `try..catch`
+
+What message will `console.log` output?
+
+```js
+try {
+  setTimeout(function() { 
+    throw new Error('Something is going wrong..')
+  }, 1000)
+} catch (error) {
+  console.log(error)
+}
+```
+
+There will be `Uncaught Error: Something is going wrong..`
+
+### 💡 Explanation:
+
+Exceptions are only a synchronous mechanism.
```

Comment:
True even for async await:
```js
asyncThrow = () => new Promise(_=> {
    setTimeout(() => {
        throw 'up';
    }, 500)
});
(async () => {
    try {
        await asyncThrow();
    }catch(ex){
        console.log('throw', ex);
    }
})()
// -> will output 'Uncaught up'
``` 
But rejected promises (still asynchronous code) can be caught
```js
asyncRej = () => new Promise((res, rej)=> {
    setTimeout(() => {
        rej('up');
    }, 500)
});

(async () => {
    try {
        await asyncRej();
    }catch(ex){
        console.log('caught', ex);
    }
})() 
// will also output 'caught up'
```